tools/CheckH5.py

```diff
@@ -0,0 +1,207 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import unittest
+import numpy as np
+import sys
+import yaml
```

Comment:
this adds the yaml module as a dependency (or maybe it was previously through something else?), but in either case that needs to be added to the installation instructions